### DISCUSSION TITLE
How do I delete personal bests

### DISCUSSION THREAD

**Discussion by DatGamerYoloYT** [Leather128/LeathersFunkinEngine#38] [Q&A] [answered]
im too fond with psych engine and I don't know how to erase personal bests on songs

**Comment by daisukidaisy** [comment]
This can only be done by deleting a file, unfortunately.
Go into (your PC user name)/AppData/Roaming/leather128, follow a few more folders and delete "leathersfunkinengine.sol"

**Comment by Leather128** [comment, marked as answer]
as of the most recent commits, this feature was added in. just press the [RESET] key to reset song or story week progress.
